pokete_classes/fightmap.py

```diff
@@ -174,6 +184,23 @@ def rechar_atk_info_box(self, attack_obs):
             self.atk_info_box.label.width + 4
         )
 
+    def rechar_atk_info_with_effect_info(self, attack_obs):
+        """Rechars the attack info box with effect info
+        ARGS:
+            attack_obs: The current attack obs"""
+        self.atk_info_box.name_label.rechar(self.effect_info_text)
+        self.atk_info_box.info_label.rechar(
+            f"{Action.SCREEN_SWITCH.mapping}:{self.attack_info_text}")
+        current_attack = attack_obs[self.box.index.index]
+
+        self.atk_info_box.label.rechar(
+            liner(self.effects_dictionary[current_attack.effect].desc, 37),
+            esccode=self.effects_dictionary[current_attack.effect].color
+        )
+        self.atk_info_box.resize(
+            self.atk_info_box.label.height + 2,
+            self.atk_info_box.label.width + 4
+        )
     def get_attack(self, attack_obs):
```

Comment:
```suggestion

    def get_attack(self, attack_obs):
```

I would add a newline here.